src/lib/Components/StickyTabPage/StickyTabPage.tsx

```diff
@@ -34,6 +35,22 @@ export const StickyTabPage: React.FC<{
   const [headerHeight, setHeaderHeight] = useState<null | number>(null)
   const tracking = useTracking()
   const headerOffsetY = useAnimatedValue(0)
+  const viewControllerRef = useRef(null)
+
+  const shouldHideBackButton = Animated.lessOrEq(headerOffsetY, -10)
+
+  Animated.useCode(
+    () =>
+      Animated.onChange(
+        shouldHideBackButton,
+        Animated.call([shouldHideBackButton], ([shouldHide]) => {
+          if (viewControllerRef.current) {
```

Comment:
Ah, gotcha. So since we don't need the reference to `self` as a `UIViewController`, do we need the `viewControllerRef` at all?

Reply:
oh nope, good catch